src/services/user-token.ts

```diff
@@ -0,0 +1,21 @@
+import { parseQueryString } from './query-string'
+
+import { MiscTypes } from '@oneblink/types'
+let userToken: string | MiscTypes.NoU = null
+
+export function setUserToken(token: string | MiscTypes.NoU): void {
+  userToken = token || null
+}
+
+export function getUserToken() {
+  if (userToken) {
+    return userToken
+  }
+  return getQueryStringToken()
```

Comment:
I dont think we should default to this, its something that should be in the application using this SDK IMO.